public/templates/partials/doc-annotations.html

```diff
@@ -53,7 +53,7 @@ <h2 translate="document.label.annotationdiscussion"></h2>
       </blockquote>
 
       <div class="comment-body"
-        ng-class="{ highlight: annotation.id === annotationId }">
+        ng-class="{ highlight: !(subCommentId) && (annotation.id-0) === (annotationId-0) }">
```

Comment:
Angular has no `parseInt` in templates, so the only way to type juggle is by forcing the values to `int`s using native methods.  Since `-` is only a valid operator for numbers in JS (unlike `+`), subtracting zero is an easy way to juggle to `int`.  (And here we've got mixed strings and ints that are derived, so we have to juggle.)